CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * [#663](https://github.com/clojure-emacs/cider-nrepl/pull/663): Fix non-recognized `recur` inside `case` in the debugger.
 * [#664](https://github.com/clojure-emacs/cider-nrepl/pull/664): Fix continue-all in conditional break.
 * [#665](https://github.com/clojure-emacs/cider-nrepl/pull/665): Fix form location in map literals.
+* [#666](https://github.com/clojure-emacs/cider-nrepl/pull/666): Add a check in apropos var-query-map.
```

Comment:
This should be under `master`. Add a new "Bugs fixed" section there, as 0.24 was release a few days ago.